prac_07/project_management.py

```diff
@@ -95,8 +100,16 @@ def add_new_project(projects):
     projects.append(project)
```

Comment:
I thought this is the update_project function. Do you need to append the project?